Makefile

```diff
@@ -13,6 +13,10 @@ lint-all: lint audit udeps
 audit:
 	cargo audit
 
+install-deps:
+	apt-get update -y
+	apt-get install --no-install-recommends -y build-essential clang protobuf-compiler ocl-icd-opencl-dev cmake
```

Comment:
I added all `install-deps` initially because it was getting used in `lint-all` check
 https://github.com/ChainSafe/fil-actor-states/blob/cc95396a10ff31d1141f96c0b258a71e7abe4231/.github/workflows/tests.yml#L67
 
 looks like its not actually used so removed it now